serve.js

```diff
@@ -16,26 +17,18 @@ const args = require('minimist')(process.argv.slice(2));
 const compression = require('compression');
 const child_process = require('child_process');
 // -------------------------------------------
-
-if (args.hasOwnProperty('help') || args._.includes('help')) {
-    console.log(`Usage: ./serve.js [OPTION]...
-Starts the Aspine web server.
-
-Options:
-  --insecure  do not secure connections with TLS (HTTPS)
-  --help      display this help and exit
-`);
-    process.exit();
-}
+program
+    .version('2.6.1')
+    .option('-i, --insecure', 'do not secure connections with TLS (HTTPS)')
+    .option('-p, --port <number>', 'port to listen on', 8080)
+    .parse();
 
 // ------------ Web Server -------------------
-const port = 8080;
-
 const app = express();
-app.use(compression({ filter: (..._) => true }));
-app.listen(port, () => console.log(`Aspine listening on port ${port}!`));
+app.use(compression());
+app.listen(program.port, () => console.log(`Aspine listening on port ${program.port}!`));
 
-if(!(args.hasOwnProperty('insecure') || args._.includes("insecure"))) {
+if(!program.insecure) {
```

Comment:
This breaks the usage of `npm start insecure`, meaning that we need to update documentation to use `npm start -- -i` instead (which IMO is not very elegant).

Reply:
no, -i or --insecure

Reply:
My main concern is not inelegance but that documentation needs to be updated to use `npm start -- -i`.